src/satosa/backends/oauth.py

```diff
@@ -178,6 +182,12 @@ class FacebookBackend(_OAuthBackend):
     Backend module for facebook.
     """
 
+    """
+    The default graph endpoint is for backward compatibility with previous versions of the
+    Facebook backend in which the graph endpoint was hardcoded in the code.
+    """
+    DEFAULT_GRAPH_ENDPOINT = "https://graph.facebook.com/v2.5/me"
```

Comment:
I think this line here also addresses issues #73 #72 
So maybe these should be closed or at least be reworded ?